docs/devel/testing.md

```diff
@@ -112,43 +141,47 @@ Currently, collecting coverage is only supported for the Go unit tests.
 To run all unit tests and generate an HTML coverage report, run the following:
 
 ```sh
-cd kubernetes
 KUBE_COVER=y hack/test-go.sh
 ```
 
-At the end of the run, an the HTML report will be generated with the path printed to stdout.
+At the end of the run, an the HTML report will be generated with the path
+printed to stdout.
 
-To run tests and collect coverage in only one package, pass its relative path under the `kubernetes` directory as an argument, for example:
+To run tests and collect coverage in only one package, pass its relative path
+under the `kubernetes` directory as an argument, for example:
 
 ```sh
-cd kubernetes
 KUBE_COVER=y hack/test-go.sh pkg/kubectl
 ```
 
-Multiple arguments can be passed, in which case the coverage results will be combined for all tests run.
+Multiple arguments can be passed, in which case the coverage results will be
+combined for all tests run.
 
-Coverage results for the project can also be viewed on [Coveralls](https://coveralls.io/r/kubernetes/kubernetes), and are continuously updated as commits are merged. Additionally, all pull requests which spawn a Travis build will report unit test coverage results to Coveralls. Coverage reports from before the Kubernetes Github organization was created can be found [here](https://coveralls.io/r/GoogleCloudPlatform/kubernetes).
+Coverage results for the project can also be viewed on
+[Coveralls](https://coveralls.io/r/kubernetes/kubernetes), and are
+continuously updated as commits are merged. Additionally, all pull requests
+which spawn a Travis build will report unit test coverage results to
```

Comment:
Not exactly part of your PR, but I don't think this is true anymore since we shut off Travis per #23611

No idea whether coveralls is being kept up to date, but it doesn't look like it